src/pages/home/report/ReportActionsView.js

```diff
@@ -49,7 +49,7 @@ const propTypes = {
 };
 
 const defaultProps = {
-    reportActions: {},
+    reportActions: [],
```

Comment:
Do we need to update other usages of reportActions? e.g. [here](https://github.com/Expensify/App/blob/4512136a17062147ce0e38b7029d7769cd903a34/src/components/ReportActionItem/IOUAction.js#L64)

Reply:
Cool, thanks for clarifying!